src/views/List.jsx

```diff
@@ -1,16 +1,48 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 /** List component that displays items in a user's shopping cart  */
 export function List({ data }) {
+	const [filteredList, setFilteredList] = useState(null);
+	const [filterInput, setFilterInput] = useState(null);
+
+	/* Use handler to change the state of filterInput  and convert all items to lowercase to facilitate a more thorough search */
+	const handleInput = (event) => {
+		const value = event.target.value.toLowerCase();
+		setFilterInput(value);
```

Comment:
This is easy to read and follow, and if you'd like you can simplify it further with 

> `setFilterInput(e.target.value.toLowerCase());`